File: src/test/java/com/kobylynskyi/graphql/codegen/kotlin/compile-pass/EventsByCategoryAndStatusQueryResolver.kt

/**
 * List of events of a specified category.
 */
interface EventsByCategoryAndStatusQueryResolver {

    /**
     * List of events of a specified category.
     */
    @Throws(Exception::class)
    fun eventsByCategoryAndStatus(categoryId: String, status: EventStatus?): List<Event?>

}

File: src/test/java/com/kobylynskyi/graphql/codegen/scala/compile-pass/EventsByCategoryAndStatusQueryResolver.scala
import EventStatus._

/**
 * List of events of a specified category.
 */
trait EventsByCategoryAndStatusQueryResolver {

    /**
     * List of events of a specified category.
     */
    @throws[Exception]
    def eventsByCategoryAndStatus(categoryId: String, status: EventStatus): scala.Seq[Event]

}
